### DISCUSSION TITLE
Running ADF for a cam baseline with different history files (h0) than the diagnostics run (h0a)

### DISCUSSION THREAD

**Discussion by adagj** [NCAR/ADF#325] [Q&A] [answered]
Hi,
I want to make a comparison between simulations using a recent CAM version and some older simulations. The newer simulations have h0a history files, while the older simulations have h0 history files. Is there a way to set that in ADF? I tried to add hist_str to both diag_cam_climo and diag_cam_baseline_climo but that didn't seem to work. Feedback highly appreciated :-)

**Comment by bitterbark** [comment, marked as answer]
Are you using the latest version ? Hist_str should be set in diag_cam_climo and baseline, not basic_info anymore, with the purpose of having different file names possible. On Aug 29, 2024, at 06:24, Ada Gjermundsen ***@***.***> wrote:﻿
Hi,
I want to make a comparison between simulations using a recent CAM version and some older simulations. The newer simulations have h0a history files, while the older simulations have h0 history files. Is there a way to set that in ADF? I tried to add hist_str to both diag_basic_info and diag_cam_climo but that didn't seem to work. Feedback highly appreciated :-)

**Reply by adagj** [reply]
That worked! Thank you so much :)
